package-inspector/src/main/res/menu/options_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@+id/search"
+        android:title="Search Pkgs"
+        android:icon="@android:drawable/ic_search_category_default"
+        app:showAsAction="collapseActionView|ifRoom"
+        app:actionViewClass="android.widget.SearchView" />
+</menu>
```

Comment:
new line at end of file

Reply:
d075004e99b5b1eede33944d1832c3f8015e420c